tools/doc/html.js

```diff
@@ -118,7 +118,7 @@ function render(opts, cb) {
 
     template = template.replace(/__ID__/g, id);
     template = template.replace(/__FILENAME__/g, filename);
-    template = template.replace(/__SECTION__/g, section);
+    template = template.replace(/__SECTION__/g, section || 'Index');
```

Comment:
The reason that I did use "Index" is [here](https://github.com/nodejs/node/blob/master/doc/template.html#L27), should we unify them? any thoughts @jasnell?